docs/purchasing/parts.json

```diff
@@ -134,5 +134,90 @@
         "url": "https://www.amazon.com/Inch-XHF-Waterproof-Insulation-Wear-Resistant/dp/B0919F4YFZ"
       }
     ]
+  },
+  {
+    "pn": "ec-microfit-f-wire-1x3",
+    "notes": "Micro-Fit 3.0 Female Wire Housing 1x3",
+    "manufacturer": "Molex",
+    "mpn": "0436450300",
+    "avg_price": 0.49,
+    "price_unit": "",
+    "sources": [
+      {
+        "supplier": "Digikey",
+        "spn": "WM1846-ND",
+        "price": 0.49,
+        "price_units": 1,
+        "url": "https://www.digikey.com/en/products/detail/molex/0436450300/268975"
+      }
+    ]
+  },
+  {
+    "pn": "ec-microfit-f-crimp-20-24awg",
```

Comment:
I would maybe keep our internal part numbers shorter and less esoteric. When naming imported components in the CAD directories, my rule of thumb is:

Are we likely to ever have 2 parts that are alike in every way other than this one diameter measurement or feature detail? 
* if yes, then it's ok to append it to file name / part descriptor
* if no, then it serves no purpose

So detailed enough to get the gist of what it is and differentiate, not to encode as much info as possible.

Reply:
Yeah, we will definitely have male (m) and female (f) crimps and very likely also multiple sizes of these.  'crimp' probably also needs to be defined because both housings and connectors go under the ec- prefix.  If we split ec- (connectors) and ep- (pins) we could probably drop -crimp-.

Also, there are so many different options when ordering these contacts, and the wire gauge and gender are the ones I've seen most frequently mixed up by purchasing (not here, but at previous companies), so I put them in the pn.